### DISCUSSION TITLE
对mkv格式的视频文件不能解码播放

### DISCUSSION THREAD

**Discussion by GinChow** [messense/aliyundrive-webdav#458] [Q&A] [answered]
目前对解码视频格式有限制吗？为什么阿里云盘官方接口可以播放mkv格式，本地云盘却不能播放，是不是还没有集成对应的解码器？

**Comment by messense** [comment, marked as answer]
你的播放器问题吧，本项目不会对任何视频文件做转码解码操作都是原片格式，不同的格式需要播放器支持。

**Reply by GinChow** [reply]
了解，谢谢

**Reply by xueyitan** [reply]
那阿里播放器可以让我选择播放器么，因为我本地有一个ommiplayer播放器，支持大多数格式

**Comment by llx7878** [comment]
alist那个为啥没法播放mkv
